docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+
+services:
+  web:
+    build:
+      context: .
+    volumes:
+      - .:/app
+    ports:
+      - "3000:3000"
+    command: bundle exec rails s -p 3000 -b '0.0.0.0'
+    depends_on:
+      - db
+
+  db:
+    build:
+      context: .
+      dockerfile: docker/postgres/Dockerfile
+    ports:
+      - "54321:5432"
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: trust
+      POSTGRES_DB: openstreetmap
+    volumes:
+      - ./docker-db-data:/var/lib/postgresql/data
```

Comment:
This should really be a data volume, rather than a file mapping.
See: https://docs.docker.com/compose/compose-file/compose-file-v2/#volume-configuration-reference